src/edfi_amt_data_lake/api/changeVersion.py

```diff
@@ -63,6 +80,15 @@ def get_change_version_updated(school_year) -> bool:
     oldestChangeVersion = ''
     newestChangeVersion = ''
 
+    disable_change_version = config('DISABLE_CHANGE_VERSION', default=False)
+
+    if disable_change_version is True:
```

Comment:
This condition is not working. If the flag is True it doesn't enter to this block.

Reply:
Done.